src/main/java/frc/robot/auto/generator/PathPlannerAutoGenerator.java

```diff
@@ -50,69 +71,124 @@ public PathPlannerAutoGenerator(DriveSubsystem drive, ArmSubsystem arm, ShooterS
             drive::getRobotRelativeSpeeds,
             drive::setRobotRelativeChassisSpeeds,
             new HolonomicPathFollowerConfig(
-                new PIDConstants(Constants.PathPlanner.TRANSLATION_P, Constants.PathPlanner.TRANSLATION_I, Constants.PathPlanner.TRANSLATION_D),
-                new PIDConstants(Constants.PathPlanner.ROTATION_P, Constants.PathPlanner.ROTATION_I, Constants.PathPlanner.ROTATION_D),
-                Constants.PathPlanner.MAXIMUM_VELOCITY, 
-                Constants.PathPlanner.DRIVEBASE_RADIUS, 
+                new PIDConstants(Constants.Auto.TRANSLATION_P, Constants.Auto.TRANSLATION_I, Constants.Auto.TRANSLATION_D),
+                new PIDConstants(Constants.Auto.ROTATION_P, Constants.Auto.ROTATION_I, Constants.Auto.ROTATION_D),
+                Constants.Auto.MODULE_MAXIMUM_VELOCITY, 
+                Constants.Auto.DRIVEBASE_RADIUS, 
                 new ReplanningConfig(true, false)
             ),
             AllianceUtil::isRedAlliance,
             drive
         );
 
-        autoSelector = AutoBuilder.buildAutoChooser(); // the sendablechooser should really only be the name strings (fix w/ jake later)
-        getTab().add("Auto Selection", autoSelector).withPosition(0, 0).withSize(2, 1);
 
+        autoSelector = AutoBuilder.buildAutoChooser(); // the sendablechooser should really only be the name strings (fix w/ jake later)
```

Comment:
Change sendablechooser to be populated with the auto names (can probably steal code to do this from pathplannerlib)

Reply:
https://github.com/mjansen4857/pathplanner/blob/d0e12f59430b869bde70180cb709e175b762fc67/pathplannerlib/src/main/java/com/pathplanner/lib/auto/AutoBuilder.java#L676 `getAllAutoNames` is a public method